src/openseadragon.js

```diff
@@ -1403,6 +1405,26 @@ function OpenSeadragon( options ){
             CHROMEEDGE: 7
         },
 
+        /**
+         * Keep track of which {@link Viewer}s have been created.
```

Comment:
This is a private member, so it should be marked as such using `@private`.

Reply:
I've marked it as `@private`